src/libs/conduit/conduit_node.cpp

```diff
@@ -11962,6 +11962,33 @@ Node::children() const
     return NodeConstIterator(this);
 }
 
+//---------------------------------------------------------------------------//
+Node&
+Node::add_child(const std::string &name)
+{
+    Schema &child_schema = m_schema->add_child(name);
+    Schema *child_ptr = &child_schema;
+    Node *child_node = new Node();
+    child_node->set_schema_ptr(child_ptr);
+    child_node->m_parent = this;
+    m_children.push_back(child_node);
+    return  *m_children[m_children.size() - 1];
+}
+
+//---------------------------------------------------------------------------//
+const Node&
+Node::get_child(const std::string &name) const
+{
+    if(!m_schema->has_direct_child(name))
+    {
+        CONDUIT_ERROR("Cannot fetch non-existent "
+                      << "child \"" << name << "\" from Node("
+                      << this->path()
+                      << ")");
+    }
+    size_t idx = (size_t)m_schema->child_index(name);
+    return *m_children[idx];
+}
 
```

Comment:
we will also need a non-const implementation, I think thats issue the CI is pointing out

Reply:
Can do! I expect logic to be identical between the two since it's only a getter. Given that, do you have preference for the implementation? I know the brief way is to have the non-const implementation temporarily const-cast then call the const function, _Effective C++_ style, but other projects prefer having the two full methods.

Reply:
Let's go with two methods, since they are pretty short.  Thanks!

Reply:
Done!